cfme/tests/services/test_generic_service_catalogs.py

```diff
@@ -110,7 +110,7 @@ def test_bundles_in_bundle(catalog_item):
 def test_delete_dialog_before_parent_item(catalog_item):
     service_dialog = ServiceDialog(label=catalog_item.dialog)
     service_dialog.delete()
-    flash.assert_message_match(("Dialog \"{}\": Error during 'destroy': Dialog cannot be deleted " +
+    flash.assert_message_match(("Dialog \"{}\": Error during delete: Dialog cannot be deleted " +
```

Comment:
Looking at PRTs results it looks like it has not, this will need version picking